umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso19115_2.clj

```diff
@@ -234,6 +238,7 @@
       :TilingIdentificationSystems (tiling/parse-tiling-system md-data-id-el)
       :TemporalExtents (or (seq (parse-temporal-extents doc extent-info md-data-id-el))
                            (when sanitize? su/not-provided-temporal-extents))
+      :CollectionDataType (value-of (select doc collection-data-type-xpath) ".")
```

Comment:
Yes it is.  We keep the value that the user entered into the field and we return a warning message that includes all of the valid values. Yes, the generated tests produce collection data types as well as the example_data/iso19115/ISOExample-project.xml